tests/test_query.py

```diff
@@ -97,6 +103,66 @@ async def test_returning_all_fields_and_relations_with_conditions(
 
         assert_authors_ordered([actual_author], [expected_author])
 
+    @mark.parametrize(
+        argnames=["columns", "invert_selection", "get_expected_author"],
+        argvalues=[
+            (
+                None,
+                False,
+                lambda author: author.copy(include={"uuid"}),
+            ),
+            (
+                ["name"],
+                False,
+                lambda author: author.copy(include={"name"}),
+            ),
+            (
+                ["name"],
+                True,
+                lambda author: author.copy(exclude={"name", *AUTHOR_RELATIONS}),
+            ),
+            (
+                [],
+                True,
+                lambda author: author.copy(exclude=AUTHOR_RELATIONS),
+            ),
+            (
+                None,
+                True,
+                lambda author: author.copy(exclude=AUTHOR_RELATIONS),
+            ),
+        ],
+        ids=[
```

Comment:
this is an optional argument for the `@parameterize` annotation. If you omit the this arg, the test cases are printed with the str value of the passed parameter - not very readable in case you have lambda functions or python objects as parameters:
```sh
test_some_parameterized_function[<object abc123>] - PASSED
test_some_parameterized_function[<object def456>] - PASSED
```

so, the `ids` make parameterized tests look prettier:
```sh
test_some_parameterized_function[with default value] - PASSED
test_some_parameterized_function[with complex value] - PASSED
```